igvm/hypervisor_preferences.py

```diff
@@ -47,17 +60,24 @@ def __call__(self, vm, hv):
         )
         remaining_size = total_size - vms_size - self.reserved
 
-        return remaining_size < vm.dataset_obj[self.vm_attribute]
+        # does not fit at all
+        vm_size = vm.dataset_obj[self.vm_attribute]
+        if remaining_size < vm_size:
+            return False
 
+        # normalize the resource usage
+        return 1 - (vm_size / remaining_size)
```

Comment:
If there is a slight chance that this becomes a integer we should better cast it to float.